src/order/rel_iso/basic.lean

```diff
@@ -303,25 +303,42 @@ protected theorem well_founded : ∀ (f : r ↪r s) (h : well_founded s), well_f
 protected theorem is_well_order : ∀ (f : r ↪r s) [is_well_order β s], is_well_order α r
 | f H := by exactI {wf := f.well_founded H.wf, ..f.is_strict_total_order}
 
+/-- `quotient.mk` as a relation homomorphism between the relation and the lift of a relation. -/
+@[simps] def _root_.quotient.mk_rel_hom {s : setoid α} {r : α → α → Prop} (H) :
+  r →r quotient.lift₂ r H :=
+⟨@quotient.mk α _, λ _ _, id⟩
+
 /-- `quotient.out` as a relation embedding between the lift of a relation and the relation. -/
-@[simps] noncomputable def _root_.quotient.out_rel_embedding [s : setoid α] {r : α → α → Prop} (H) :
+@[simps] noncomputable def _root_.quotient.out_rel_embedding {s : setoid α} {r : α → α → Prop} (H) :
```

Comment:
I think arguably `[]` was better, as it matches how `lift₂` is defined. Is this change actually needed to make the rest of this PR work?

Reply:
Yes. `antisymm_rel.setoid` is not an instance.

Reply:
I pushed some primed versions of the lemmas using `{}` to match the `quotient.mk` / `quotient.mk'` distinction.

bors d+